google-cloud-tools-plugin/src/com/google/gct/idea/debugger/CloudDebugGlobalPoller.java

```diff
@@ -135,7 +137,9 @@ public synchronized void stopBackgroundListening() {
 
   private void queryServerForBreakpoints(CloudDebugProcessState state, Debugger client)
       throws IOException {
-
+    if (state.getDebuggeeId() == null) {
```

Comment:
Normally it should not. However you can create a state object with null fields and since it is not annotated with @notnull IJ gives a warning about it, that was the reason I've added this check.